integration-tests/resources/wallets/transfer_from_pos_vault.rho

```diff
@@ -0,0 +1,21 @@
+new return, rl(`rho:registry:lookup`), poSCh, stdout(`rho:io:stdout`), 
+     deployerId(`rho:rchain:deployerId`), transferRetCh in {
+   rl!(`rho:rchain:pos`, *poSCh) |
+   match (
+    "%TARGET_ADDR",
+    "%AMOUNT"
+   ) {
+       (targetAddr, amount) => {
+         for(@(_, PoS) <- poSCh) {
+           @PoS!("humanTransfer", targetAddr, amount, deployerId, *transferRetCh)
```

Comment:
Sorry. The error happens because of the example in the issue.
This should be `@PoS!("humanTransfer", targetAddr, amount, deployerId, *transferRetCh) |`

The `|`  in the end is important.

Reply:
`@PoS!("humanTransfer", targetAddr, amount, *deployerId, *transferRetCh) |`

Sorry. Put `*` in front of `deployerId`. `*` is a converter to convert `Name` to `Process`.